opentelemetry-api/src/opentelemetry/propagators/composite.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import typing
+
+from opentelemetry.context.context import Context
+from opentelemetry.trace.propagation import httptextformat
+
+logger = logging.getLogger(__name__)
+
+
+class CompositeHTTPPropagator(httptextformat.HTTPTextFormat):
```

Comment:
I recall these are called "Chained" propagators in the OTEP. Not sure if anything official made it to the spec.

Can you confirm that the naming is consistent? if you haven't already.

Reply:
Confirmed it's composite propagator in the spec: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-propagators.md#composite-propagator